lib/trino-orc/src/test/java/io/trino/orc/TestOrcReaderMemoryUsage.java

```diff
@@ -78,8 +77,6 @@ public void testVarcharTypeWithoutNulls()
 
                 // StripeReader memory should increase after reading a block.
                 assertGreaterThan(reader.getCurrentStripeRetainedSizeInBytes(), stripeReaderRetainedSize);
-                // There are no local buffers needed.
-                assertEquals(reader.getStreamReaderRetainedSizeInBytes() - streamReaderRetainedSize, 0L);
```

Comment:
Could you update assertion values or assertion type (`assertGreaterThan`) itself instead? Now we don't have coverage for how much memory `SliceDictionaryColumnReader` retains.

Reply:
done, replace `assertEquals ` with `assertGreaterThanOrEqual`